test/NuGet.Clients.Tests/NuGet.CommandLine.Test/NuGetSourcesCommandTest.cs

```diff
@@ -348,6 +348,47 @@ public void SourcesCommandTest_DisableSource()
             }
         }
 
+        [Fact]
+        public void SourcesCommandTest_ListSource()
```

Comment:
The class name is already `NuGetSourcesCommandTest`, using it as a prefix for the method name doesn't add anything.

Damon usually advocates for the test naming convention `<MemberUnderTest>_<Condition>_<Expectation>`, at least for unit tests. These integration/functional tests are more tricky, because there isn't a clear member being tested, it's rather a scenario.

Maybe `SourcesListDefaults_UsesDetailedFormat` would be a better name?

Reply:
Yes, good point. I did that for all the new tests on the dotnet.exe side of the house...but wasn't messing with these.
Plan to change to:  SourcesList_WithDefaultFormat_UsesDetailedFormat